Package.swift

```diff
@@ -6,7 +6,7 @@ import PackageDescription
 let package = Package(
     name: "Knit",
     platforms: [
-        .macOS(.v13),
+        .macOS(.v14),
```

Comment:
It might be somewhat nice to support back one version of macOS, was there an API you needed?

Reply:
Yep `error: 'assumeIsolated(_:file:line:)' is only available in macOS 14.0 or newer`. Technically we don't actually use that in the knit code, it's part of the additions that are used by the app but due to the way the package is setup it will fail the build.

Reply:
If we move the register functions into `Swinject` then we can drop back down.

Reply:
Ok yeah let's bump the min platform version 👍